crates/iota-grpc-api/Cargo.toml

```diff
@@ -23,6 +23,15 @@ iota-types.workspace = true
 tonic-build.workspace = true
 
 [dev-dependencies]
+# external dependencies
+anyhow.workspace = true
+
+# internal dependencies
 iota-config.workspace = true
 move-core-types.workspace = true
+simulacrum.workspace = true
 test-cluster.workspace = true
+
+[[example]]
+name = "simulacrum_grpc_workflow"
+path = "../../examples/simulacrum/grpc/complete_grpc_workflow.rs"
```

Comment:
There's a crate for that example. Surely we don't need to duplicate the work. I'd just mention the example crate in README.

Reply:
Agreed. I removed it and added the binary section in the toml file for the example directly at [f7c61bd](https://github.com/iotaledger/iota/pull/8019/commits/f7c61bdd6d079cf6826c604182129f0e9fd98c15), and I also enhanced the doc at [6741e6f](https://github.com/iotaledger/iota/pull/8019/commits/6741e6fc295517f93f4f48805b3a072dbb32026d).